guides/common/modules/con_acd_introduction.adoc

```diff
@@ -0,0 +1,113 @@
+
+|Advantages
+|full control over each host
+|deploy a number of hosts at once; self service for technically less inclined; scale vertically and/or horizontally
+
+|Disadvantages
+|extra abstraction layer between a host and its purpose
+|dependency on/limitation to existing application templates
+|===
+
+[id="{context}_usage_example"]
+== Usage Example
+
+The following application centric deployment example illustrates the deployment of hosts in an application centric approach.
+The web application requires an HAProxy, an Apache web server, a Tomcat application server, and a database server.
+
+Using ACD, this web application can be deployed and configured in a single step:
+
+. Import the Ansible playbook
++
+Upload the Ansible playbook to your {ProjectServer}.
+Specify the path and the name of the Ansible playbook via management UI.
+. Create an application definition
++
+This example requires four different services: proxy servers, web servers, application servers, and database servers.
+Those hosts are ideally part of a host group.
+. Optional: Customize Ansible variables
++
+You can either use the default values, change values, or lock the Ansible variables from being edited for an application instance.
+Ansible variables are read from the group variables of the Ansible playbook.
+. Optional: Overwrite variables from the host group
++
+For example, change the compute profile or set a different lifecycle environment.
+. Deploy an application instance
++
+** You may simply deploy the application instance;
+** run the Ansible playbook as is;
+** or customize the Ansible playbook before running it.
+
+[NOTE]
+====
+If you want to install a package using an Ansible playbook and the host is subscribed to {ProjectServer}, ensure that the package exists in the content view.
+====
+
+[id="{context}_key_terms"]
+== Key Terms
+
+.ACD
+ACD stands for _Application_ _Centric_ _Deployment_ and describes a new approach to provision and configure hosts based on the service they provide.
+
+.Ansible Playbooks
+xref:{context}_ansible_playbooks[Ansible playbooks] are the core of the application definition.
+They connect the different services and provide a whole application with one click of a button.
+These playbooks contain roles, which in turn contain tasks to properly configure hosts.
+
+.Application Definitions
+xref:{context}_application_definitions[Application definitions] are blueprints of how to deploy an application and its services.
+
+.Application Instances
+xref:{context}_application_instances[Application instances] are configured application definitions.
+You can define how many hosts to deploy per service and choose unique host names.
+
+There might be different configurations based on a testing or production environment or its expected usage.
+For example, deploying a wiki platform for 20 users might use the `1-small` compute profile, whereas hundreds of expected users might require the `3-large` compute profile.
+
+.Service
```

Comment:
I know you explained this, but I can't help but think this is going to cost ATIX in customer support. 
Any other name would help distinguish things and make thing clearer for the user and for your customers. 
I've seen poor naming conventions like this cause confusion for customers in different projects throughout the years. 
I just don't understand why this is the only name there could be for " a type of host necessary to run an application". 
I won't block it going in docs, and this is the last time I'll say anything but we can definitely do better here.

Reply:
Off the top of my head, I can't think of any other word for "a group of hosts of a type that collectively provide a single service", can you provide some ideas?

Reply:
I talked with Tomer yesterday and he suggested "Service Provider" and I would further suggest "Service-providing host" or something like that.

Reply:
We've discussed this internally once more.

We want to keep the term `service`.

Reply:
> We've discussed this internally once more.

In the spirit of our open source project, I would like to follow the logic of your discussions, because right now, I don't understand the _why_ behind this decision. 
Can you help me?

Reply:
I can change line 104 to `A service is a component of an application running on one to many hosts of the same type.`

When creating an application definition, a service is defined from a host group and Ansible group.
When creating an application instance, you must assign hosts to services. According to the service counts of the application definition, one service is provided by zero to many hosts.

Reply:
Hi @maximiliankolb that sounds like an improvement. 
I don't want to hold up this any longer. We can see if this issue comes up anywhere else in the community, and if so, we can revisit. 
Thanks for your patience.